pkg/postgres/repository.go

```diff
@@ -399,6 +416,58 @@ ORDER BY estimates.identifier ASC`,
 	return
 }
 
+//GetRecentlyActiveRuns collects all the data about any runs that have been updated
+//since touchThreshold. The Run#Finished method can be used to determine which runs
+//have arrived at their terminal station, and can therefore be removed from state.
+func (a *RepositoryAgent) GetLatestEstimates(station martaapi.Station) (res []LastestEstimate, err error) {
+	rows, err := a.DB.Query(`
+WITH station_estimates AS (
+  SELECT runs.run_group_identifier,
+    estimates.estimated_arrival_time,
+    estimates.estimate_moment,
+    ROW_NUMBER() OVER (PARTITION BY runs.run_group_identifier
+      ORDER BY estimates.estimate_moment DESC) AS rank
+  FROM arrivals
+  JOIN estimates
+    ON estimates.arrival_identifier = arrivals.identifier
+  JOIN runs
+    ON arrivals.run_identifier = runs.identifier
+  WHERE arrivals.station = 'GEORGIA STATE STATION'
+    AND arrivals.arrival_time IS NULL)
```

Comment:
Filter to the specific station (woops! I need to fix that hard-coded example before merging) and also filter out any arrivals that have an actual arrival time, since those are the ones from the past